src/plugins/intel_gpu/include/intel_gpu/graph/network.hpp

```diff
@@ -249,5 +259,14 @@ struct network {
     void check_names();
     void add_default_output_chains();
     output_chains_map::iterator add_output_chain(std::shared_ptr<primitive_inst>& p_inst);
+
+    std::unique_ptr<kernels_cache> _kernels_cache;
+    // TODO: Move from cldnn::program to cldnn::network for multi-thread issue.
+    // Need to consider the location of these caches for more optimizations.
```

Comment:
I think this comment is redundant at this point.

Reply:
Done